File: AlgoRunner.Api/AlgoRunner.Api/Entities/Project.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AlgoRunner.Api.Entities
{
    public class Project
    {
        public int Id{ get; set; }
        public Activity Activity { get; set; }
        public string Name { get; set; }
        public string Desc { get; set; }
        public string CreatedBy { get; set; }
        public bool IsFavorite { get; set; }
        public DateTime LastExecutionDate { get; set; }
        public List<ProjectExecution> ExecutionsList { get; set; }
        public List<Algorithm> AlgorithmsList { get; set; }
    }
}

File: AlgoRunner.Web/src/app/layout/projects-detail/models/project.ts
import {ProjectExecution} from './projectExecution';
import { Algorithm } from '../../algorithm/models/algorithm';
import { Activity } from '../../../shared/models/activity';

export class Project {
    id: number;
    name: string;
    desc: string;
    isFavorite : boolean;
    lastExecutionDate: Date;
    projectExecutions : ProjectExecution[];
    algorithmsList : Algorithm[];
    activity: Activity;
  }